src/RewriteRules.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-zendframework-bridge for the canonical source repository
+ * @copyright Copyright (c) 2019 Laminas Foundation (https://getlaminas.org)
```

Comment:
we should update it now to refer to COPYRIGHT.md file now

Reply:
Yes, please. :)